3_strings_func.py

```diff
@@ -0,0 +1,105 @@
+import re
+
+
+# Function to convert text to lowercase
+def to_lower(text):
+    return text.lower()
+
+
+# Function to calculate the number of whitespace characters
+def count_whitespace(text):
+    return sum(1 for char in text if char.isspace())
+
+
+# Function to replace ' iz ' with ' is '
+def replace_iz_with_is(text):
```

Comment:
Possible, but the function is too specific and can be used only for the specific task in the current homework.